docs/source/tertiary/regression-tests.rst

```diff
@@ -25,34 +32,55 @@ Example
 
 .. code-block:: python
 
+  import pandas as pd
   from pyspark.ml.linalg import DenseMatrix
-  import pyspark.sql.functions as fx
+  from pyspark.sql import Row
+  from pyspark.sql.functions import col, lit
   import numpy as np
 
   # Read in VCF file
-  df = glow.transform("split_multiallelics", spark.read.format('vcf').load(path)).cache()
+  variants = spark.read.format('vcf').load(genotypes_vcf)
+
+  # genotype_states returns the number of alt alleles for each sample
+  # mean_substitute replaces any missing genotype states with the mean of the non-missing states
+  genotypes = glow.transform('split_multiallelics', variants) \
+    .withColumn('gt', glow.mean_substitute(glow.genotype_states(col('genotypes')))) \
+    .cache()
+
+  # Read covariates from a CSV file and add an intercept
+  covariates = pd.read_csv(covariates_csv, index_col=0)
+  covariates['intercept'] = 1.
 
-  # Generate random phenotypes and an intercept-only covariate matrix
-  n_samples = df.select(fx.size('genotypes')).first()[0]
-  covariates = DenseMatrix(n_samples, 1, np.ones(n_samples))
-  np.random.seed(500)
-  phenotypes = np.random.random(n_samples).tolist()
-  covariates_and_phenotypes = spark.createDataFrame([[covariates, phenotypes]],
-    ['covariates', 'phenotypes'])
+  # Read phenotypes from a CSV file
+  continuous_phenotypes = pd.read_csv(continuous_phenotypes_csv, index_col=0)
```

Comment:
nit: I think we could just call this `phenotypes`. `continuous_phenotypes` might cause some confusion about the meaning of continuous in this context.